Assignments/Program.cs

```diff
@@ -1,3 +1,81 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Spark23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T22 - Swap Indices Values
+// ---------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using static System.ConsoleColor;
 
-Console.WriteLine("Hello, World!");
+namespace Spark;
+
+public class T22 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "Index Swaping Game:-" + "\x1B[0m");
+      Console.WriteLine ("Random series of numbers will be displayed. "
+         + "The user has to enter two indeces value for the values to be swaped.");
+      PerformIndexSwap ();

```

Comment:
Get the inputs and checks are in main method after that you call PerformIndexSwap method.